src_c/time.c

```diff
@@ -309,40 +307,44 @@ clock_tick_base(PyObject *self, PyObject *arg, int use_accurate_delay)
         }
 
         if (use_accurate_delay)
-            delay = accurate_delay(delay);
+            accurate_delay(delay);
         else {
             // this uses sdls delay, which can be inaccurate.
             if (delay < 0)
-                delay = 0;
+                delay = 0.0f;
 
-            Py_BEGIN_ALLOW_THREADS;
-            SDL_Delay((Uint32)delay);
-            Py_END_ALLOW_THREADS;
+            if (delay) {
```

Comment:
Maybe this should be `if (delay > 0.01f)`?
It would be interesting to see how often the `if(delay)` block is avoided in normal use? Is it never, or often?

Reply:
Maybe that can be replaced by delay >= 1, because it’s going to be rounded to 0 if it is between 0 and 1 anyways

Reply:
I fixed that